### ISSUE TITLE
[Feature Request]: I would like to run a project without having it added to hop-config.json

### ISSUE THREAD

**Issue by hansva** [apache/hop#2916]
### What would you like to happen?

In some cases, it might be useful to be able to use and run a project without the explicit need to add it to hop-config.json.

We could add an option --project-location to point to a folder (or zip?) and it will load that as a "nameless" project and execute a workflow or pipeline in there.

This would simplify use cases where the project is managed or provided via a different system.

### Issue Priority

Priority: 3

### Issue Component

Component: Hop Run

**Comment by mattcasters** [comment]
To avoid having to add lots of options we could perhaps create this nameless project as per definition a folder location with in it contained all that is needed to run, including one or more environment (JSON) files, using some conventions.
Then we can add this `export to nameless project` feature in the UI and on the command line.

**Comment by mattcasters** [comment]
This is a variant of #2916 : if we can simply run `sh hop -pc /path/to/hop-config.json` and we'd allow relative project paths in the config, we can even ship with the hop-config.json file in the metadata to allow quick setup, even with environment files.

**Comment by hansva** [comment]
Project zip can now include metadata and environment variables. So we can use that zip as a base for execution of a project

**Comment by mattcasters** [comment]
I'm fixing that FileTree dialog right now. I want to have the option to simply have a copy as a zip file.
Then you can actually open the project zip file as-is.

**Comment by mattcasters** [comment]
I think the "Project export" was designed for filing bug reports and things like that?
Did we ever get files like that?

**Comment by hansva** [comment]
No, don't think so as it might be exposing a bit too much information to us. We do have 1 client that is using them as a stateless deployment. This ticket was to bring that functionality back to OSS as others might want to create a form of "deployments" that aren't git based but using an actual artifact.

**Comment by mattcasters** [comment]
<img width="1453" height="1194" alt="Image" src="https://github.com/user-attachments/assets/84a06aa2-7028-4605-b828-abaf848c2d34" />

Then add a project + double click on the zip file:

<img width="2203" height="1242" alt="Image" src="https://github.com/user-attachments/assets/fcf2a467-193c-419c-88d9-d231f13dcb86" />

--> disables all options in the dialog, sets it as read-only automatically:

<img width="2215" height="279" alt="Image" src="https://github.com/user-attachments/assets/1388afe0-5a61-4c6d-8410-de2a3c113076" />

**Comment by mattcasters** [comment]
So this is a new use-case. That's fine.

**Comment by mattcasters** [comment]
I rejected #2916 because it's just too tricky compared to simply setting `HOP_CONFIG_FOLDER`.
